src/solidity/decodeInfo.js

```diff
@@ -0,0 +1,353 @@
+'use strict'
+
+/**
+  * Uint decode the given @arg type
+  *
+  * @param {String} type - type given by the AST
+  * @return {Object} returns decoded info about the current type: { needsFreeStorageSlot, storageBytes, typeName, decoder}
+  */
+function Uint (type) {
+  if (type.split(' ')) {
+    type = type.split(' ')[0]
+  }
+  return {
+    needsFreeStorageSlot: false,
+    storageBytes: parseInt(type.replace('uint', '')) / 8,
```

Comment:
To make it a bit more flexible, you could also support the `uint` alias (without number) here.